code/modules/power/apc.dm

```diff
@@ -787,11 +788,6 @@
 	last_nightshift_switch = world.time
 	set_nightshift(!nightshift_lights)
 
-/obj/machinery/power/apc/run_obj_armor(damage_amount, damage_type, damage_flag = 0, attack_dir)
-	if(damage_flag == "melee" && damage_amount < 10 && (!(stat & BROKEN) || malfai))
```

Comment:
you'll probably want to keep this check though, it just won't include the melee damage and amount check.

Reply:
See that was my thought, but as far as I could tell these functionalities don't actually accomplish anything ingame, which I believe is because the syntax is messed up. This code doesn't actually work, you can't destroy a broken camera with a lower damage item, or likewise with an APC. If it can be justified feature wise I wouldn't mind doing it but otherwise my thought was to just preserve the current way they function that people are comfortable with and works well within the game.